### ISSUE TITLE
I can't download video from iPrima.cz

### ISSUE THREAD

**Issue by Cenobitax** [yt-dlp/yt-dlp#7557]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Czech republic

### Provide a description that is worded well enough to be understood

[IPrima] Downloading login page
[IPrima] Logging in
[IPrima] Downloading token
[IPrima] Extracting URL: https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/6-epizoda-6
[IPrima] 6-epizoda-6: Downloading webpage
ERROR: [IPrima] 6-epizoda-6: Unable to extract __NUXT__; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[IPrima] Extracting URL: https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/7-epizoda-7
[IPrima] 7-epizoda-7: Downloading webpage
ERROR: [IPrima] 7-epizoda-7: Unable to extract __NUXT__; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[IPrima] Extracting URL: https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/8-epizoda-8
[IPrima] 8-epizoda-8: Downloading webpage
ERROR: [IPrima] 8-epizoda-8: Unable to extract __NUXT__; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '-u', 'PRIVATE', '-p', 'PRIVATE', '-a', 'yt-dlp.exe.seznam.txt']
[debug] Batch file urls: ['https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/6-epizoda-6', 'https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/7-epizoda-7', 'https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/8-epizoda-8']
[debug] Encodings: locale cp1250, fs utf-8, pref cp1250, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[IPrima] Downloading login page
[IPrima] Logging in
[IPrima] Downloading token
[IPrima] Extracting URL: https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/6-epizoda-6
[IPrima] 6-epizoda-6: Downloading webpage
ERROR: [IPrima] 6-epizoda-6: Unable to extract __NUXT__; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 710, in extract
  File "yt_dlp\extractor\iprima.py", line 137, in _real_extract
  File "yt_dlp\extractor\common.py", line 1686, in _search_nuxt_data
  File "yt_dlp\extractor\common.py", line 1258, in _search_regex

[IPrima] Extracting URL: https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/7-epizoda-7
[IPrima] 7-epizoda-7: Downloading webpage
ERROR: [IPrima] 7-epizoda-7: Unable to extract __NUXT__; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 710, in extract
  File "yt_dlp\extractor\iprima.py", line 137, in _real_extract
  File "yt_dlp\extractor\common.py", line 1686, in _search_nuxt_data
  File "yt_dlp\extractor\common.py", line 1258, in _search_regex

[IPrima] Extracting URL: https://www.iprima.cz/serialy/wtf/wtf-welmi-trapne-ftipy/8-epizoda-8
[IPrima] 8-epizoda-8: Downloading webpage
ERROR: [IPrima] 8-epizoda-8: Unable to extract __NUXT__; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 710, in extract
  File "yt_dlp\extractor\iprima.py", line 137, in _real_extract
  File "yt_dlp\extractor\common.py", line 1686, in _search_nuxt_data
  File "yt_dlp\extractor\common.py", line 1258, in _search_regex
```

**Comment by bashonly** [comment]
Duplicate of #7229

May be fixed by #7216

**Comment by Cenobitax** [comment]
I downloaded files:

common.py
iprima.py
_utils.py

I don't know how use this files.

Pleas help me with example how use command parameters.
